data_safe_haven/administration/users/guacamole_users.py

```diff
@@ -33,49 +29,18 @@ def __init__(self, config: Config, sre_name: str, *args: Any, **kwargs: Any):
             / "postgresql"
         )
         self.sre_name = sre_name
+        self.group_name = f"Data Safe Haven Users SRE {sre_name}"
 
     def add(self, users: Sequence[ResearchUser]) -> None:
         """Add sequence of users to Guacamole"""
```

Comment:
```suggestion
```

Reply:
Sorry - I have implementations for these stubs but they aren't quite working atm.